README.md

```diff
@@ -31,6 +43,19 @@ If you'd like to contribute, please see our [contributing guide](.github/CONTRIB
 
 To find an issue to tackle, please check our [call for contributions](.github/CALL_FOR_CONTRIBUTIONS.md).
 
+We would like to leverage/outsource the Keras community not only for bug reporting,
```

Comment:
Can we care a little bit more about the label inclusivity[1]? Often external contributors start working on available bucket of freetime so generally it is important so the review timing is quite important to not accumulate too much open PRs[2].

[1]https://chaoss.community/metric-issue-label-inclusivity/
[2]https://chaoss.community/metric-review-cycle-duration-within-a-change-request/

Reply:
Very reasonable ask. Our current proposal is we will create high-priority label and low-priority label. Features / Bugs that are aligned with 6 months roadmap will be tagged high priority and the response time is 1~2 week. Not-within-roadmap will be tagged low priority and the response time is ~4 weeks.

Reply:
These are currently my (anon) stats. 
Note: this include internal team loop reviews/merge and external member in the same bucket but you know currently the internal ones are faster and it is really hard to have a pending "internal" sourced PR:

https://cauldron.io/project/5989?from_date=2021-10-17&to_date=2022-10-17&tab=performance-reviews

Reply:
Wow this is really nice! Any chance we could re-use something like this in github?

Reply:
> Any chance we could re-use something like this in github?

What do you mean? It was already just live refreshed on github public data..

If you want to select a badge see...
https://github.com/keras-team/keras-cv/issues/243#issuecomment-1083874525

Reply:
What kind of SLA does this website guarantee? Is this a one-time report, or users would need to constantly pull the data?

Reply:
P.s. you can login with your GitHub account and just fork my report with the fork button if you want to maintain your copy.

Please note that in this anon (public) version some stats like Org contribution stats (more details button)[1]  are not correct cause some internal team members are committing from non Google email accounts  (Check yourself from git log).

This will require to use the non public (non anon) version cause you need to handle some Org identities attributions.

Reply:
Thanks, this is quite helpful!

Reply:
This is super cool.  Thanks for sharing